ush/fv3gfs_remap.sh

```diff
@@ -16,7 +16,7 @@ export DATA=${DATA:-$pwd}
 export NWPROD=${NWPROD:-$pwd}
 export HOMEgfs=${HOMEgfs:-$NWPROD}
 export FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}
-export FIXfv3=${FIXfv3:-$FIX_DIR/fix_fv3_gmted2010}
+export FIXfv3=${FIXfv3:-$FIX_DIR/orog}
```

Comment:
This script is not used in the workflow.
If it is necessary, it belongs in ufs-utils.  # TODO